File: app/public_html/script/functions/samples/4.py
start = True
def loop():
    global start
    if start:
        if moveTo(12.5,12.5):
            start = False
     
    if getLowHp():
        if doYouSeeMe() and getBlockTimeLeft() <= 0:
            block()
        else:
            charge()
     
    elif not start:
        turn(50)
 

File: app/public_html/script/functions/samples/4.c
int start = 1;
loop(){
    if (start){
        if(moveTo(12.5,12.5))
            start = 0;
    }
    if(getLowHp()){
        if (doYouSeeMe() && getBlockTimeLeft() <= 0)
            block();
        else
            charge();
    }
    else if (!start)
        turn(50);
}